scripts/install/sabnzbd.sh

```diff
@@ -10,30 +10,45 @@
 #   including (via compiler) GPL-licensed code must also be made available
 #   under the GPL along with build & install instructions.
 
-user=$(cut -d: -f1 < /root/.master.info)
-password=$(cut -d: -f2 < /root/.master.info)
-distribution=$(lsb_release -is)
-codename=$(lsb_release -cs)
-#latest=$(curl -s https://sabnzbd.org/downloads | grep -m1 Linux | grep download-link-src | grep -oP "href=\"\K[^\"]+")
-latest=$(curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | grep -Po '(?<="browser_download_url":).*?[^\\].tar.gz"' | sed 's/"//g')
-latestversion=$(echo $latest | awk -F "/" '{print $NF}' | cut -d- -f2)
-
 . /etc/swizzin/sources/functions/pyenv
 . /etc/swizzin/sources/functions/utils
 
-LIST='par2 p7zip-full python3-dev python3-setuptools python3-pip python3-venv libffi-dev libssl-dev libglib2.0-dev libdbus-1-dev'
+user=$(_get_master_username)
+password=$(_get_user_password ${user})
+#latest=$(curl -s https://sabnzbd.org/downloads | grep -m1 Linux | grep download-link-src | grep -oP "href=\"\K[^\"]+")
+latest=$(curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | grep -Po '(?<="browser_download_url":).*?[^\\].tar.gz"' | sed 's/"//g')
+latestversion=$(echo $latest | awk -F "/" '{print $NF}' | cut -d- -f2)
```

Comment:
```suggestion
latest=$(https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | jq -r .tarball_url)
latestversion=$(curl https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | jq -r .tag_name)
```

Reply:
This will of course only work if the tag tarball is the same as the relase file ending in .tar.gz, so if the released source is modified compared to the repo, we should target the actual release instead

Reply:
I'd prefer to avoid `curl`ing the api twice. It takes extra time to make another request and also uses another (limited) request. The `latest` tag should have all the information we need to properly set the `latestversion` variable

Reply:
Also 
```
curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | jq -r .tarball_url
```
Does not return a `.tar.gz` link

Reply:
The `tarball.url` is literally the entire repo tar'd at time of release and is not the same as the src tar gz

![image](https://user-images.githubusercontent.com/30909703/107406382-3aa22480-6abd-11eb-8255-ccc419eb22bd.png)

Reply:
> I'd prefer to avoid `curl`ing the api twice

Sure, we can store the first response, the point was mostly to switch to using `jq` instead which is cleaner and more expectable

> Does not return a `.tar.gz` link

<img width="510" alt="afbeelding" src="https://user-images.githubusercontent.com/23618693/107406651-f5a3df80-6b08-11eb-8c28-c2e1bfc04704.png">


> The `tarball.url` is literally the entire repo tar'd at time of release and is not the same as the src tar gz

Fair enough, I'll come back to you on that with a proper `jq` filter to retrieve the released .tar.gz

Reply:
Already got it ;)

```
latest=$(curl -sL https://api.github.com/repos/sabnzbd/sabnzbd/releases/latest | jq -r '.assets[]?.browser_download_url | select(contains("tar.gz"))')
latestversion=$(echo $latest | awk -F "/" '{print $NF}' | cut -d- -f2)
```